developer-portal/for-developers/custom-integration/fast-sdks/ios/use-fast-checkout.mdx

```diff
@@ -0,0 +1,107 @@
+| `Product.platformVariantId` | String?                 | A unique sub-identifier for this product.                                                                                                                      |
+| `Product.productOptions`    | [`ProductOption`] array | Configurable properties of this Product (e.g. color, size, etc.). Refer to [ProductOption Object Breakdown](#productoption-object-breakdown) for more details. |
+| `Product.quantity`          | Int                     | The number of this item that should be purchased.                                                                                                              |
+
+#### ProductOption Object Breakdown
+
+<MobileProductOptionsWarning sdkFullName="Fast iOS SDK" />
+
+A `ProductOption` is a configuration that further describes the product being ordered (e.g. _ProductOption(id: "color", value: "blue")_).
+
+| Name                 | Type   | Description                                            |
+| -------------------- | ------ | ------------------------------------------------------ |
+| `ProductOption.id`   | String | The identifier of the product option or configuration. |
+| `ProductOptionvalue` | String | The value of the specified option.                     |
+
+## Checkout with a Cart ID
+
+Pass in a `cartId` that the Fast backend can break into a list of products and, if applicable, any discount code.
+
+### Cart Checkout Function Breakdown
+
+```swift
+/// Present the Fast Checkout flow.
+/// - cartId: Unique identifier for a user's cart.
+/// - couponCode: A discount code that the user might have entered.
+/// - callback: A callback to listen for events during checkout.
+public func checkout(
+	presentedBy: UIViewController,
+	cartId: String,
+	couponCode: String,
+	callback: @escaping (FastEvent) -> Void
+)
+```
+
+| Name          | Type             | Description                                                                                                |
+| ------------- | ---------------- | ---------------------------------------------------------------------------------------------------------- |
+| `presentedBy` | UIViewController | Content view controller associated with the checkout flow.                                                 |
+| `cartId`      | String           | The ID of the cart associated with the Fast Checkout.                                                      |
+| `couponCode`  | String           | (Optional) Coupon, if any, included during checkout.                                                       |
+| `callback`    | Closure          | Callback to notify your app about important events (e.g. when an order is created, updated, or completed). |
+
+## Initialize Theme
+
+Currently the SDK can support dark or light mode as set at the system level, but does not currently support manually toggling the theme using the SDK.
```

Comment:
Change this to:

```swift
/// Override the theme setting for Fast Checkout
/// - theme: The theme value to set (auto, light, dark)
public func setTheme(theme: FastTheme) -> Void
 ```

The theme of the SDK is by default set to `auto`, which utilizes the current device theme, but can be overridden to be `dark` or `light`.